utils/add_copyright.py

```diff
@@ -23,9 +23,11 @@
 import fileinput
 import fnmatch
 import os
+import re
 import sys
 
-COPYRIGHT = 'Copyright 2015 The Shaderc Authors. All rights reserved.'
+COPYRIGHTRE = re.compile('Copyright \d+ The Shaderc Authors. All rights reserved.')
+COPYRIGHT = 'Copyright 2016 The Shaderc Authors. All rights reserved.'
```

Comment:
Good point.  I don't know exactly how we handle that situation, but once we spell out the rules, it will be trivial to modify this RE.